assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -23,7 +23,6 @@ import styled from 'styled-components';
 /**
  * WordPress dependencies
  */
-import { Spinner } from '@wordpress/components';
```

Comment:
It's interesting to note that this <Spinner /> component from Gutenberg is not displaying anything across the plugin. We are also using this component [here](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/header/buttons.js#L159), should we remove it there as well? Or maybe replace for something more useful?

Reply:
See #790 for that.

It's very odd that nothing is being displayed, didn't realize that. This definitely wasn't always the case. Probably due to some update when they changed how the components render, and now there is no stylesheet being loaded for it.

Even more of a reason to get rid of this.